cli-qpid-jms/src/main/resources/jndi.properties

```diff
@@ -18,16 +18,15 @@
 #
 
 # Set the InitialContextFactory class to use
-#java.naming.factory.initial=org.apache.qpid.amqp_1_0.jms.jndi.PropertiesFileInitialContextFactory
 java.naming.factory.initial=org.apache.qpid.jms.jndi.JmsInitialContextFactory
 #java.naming.provider.url=src/main/resources/jndi.properties
 
 # Define the required ConnectionFactory instances
 # connectionfactory.<JNDI-lookup-name>=<URI>
-connectionfactory.amqFactory=amqp://localhost:5672
+connectionfactory.amqFactory=${AMQFACTORY}
 
 # Configure the necessary Queue and Topic objects
 # queue.<JNDI-lookup-name>=<queue-name>
 # topic.<JNDI-lookup-name>=<topic-name>
-queue.amqQueue=qeQueue
-topic.amqTopic=qeTopic
+queue.amqQueue=${AMQQUEUE}
```

Comment:
How do you inject values into these ${} params? How is this used? Or this is just a template and user should provide hist own properly filled jndi?

Reply:
It is a template that takes -D java system properties, or environment variables, so `export AMQQUEUE=whatever`. User must give the new `--conn-use-config-file` switch, only then JNDI is used.